src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -382,6 +382,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('search.experimental.quickAccess.preserveInput', "Controls whether the last typed input to Quick Search should be restored when opening it the next time."),
 			'default': false
 		},
+		'search.experimental.quickAccess.seedFileSearchStringFromSelection': {
+			'type': 'boolean',
+			'description': nls.localize('search.experimental.quickAccess.seedFileSearchStringFromSelection', "Controls whether the search string in the Search File Widget is seeded from the editor selection."),
```

Comment:
Instead of `Search File Widget`, we should use `Quick Open`.

Reply:
Got it will change